x-pack/plugin/esql/qa/testFixtures/src/main/resources/enrich.csv-spec

```diff
@@ -31,6 +31,105 @@ FROM sample_data
 median_duration:double | env:keyword
 ;
 
+withQualifierConflict
+required_capability: enrich_load
+required_capability: enrich_with_qualifier
+ROW client_ip = "172.21.0.5", env = 1
+| ENRICH clientip_policy ON client_ip AS c
```

Comment:
We don't use `AS` anywhere and instead are using `=` (see STATS).
The command above would be `ENRICH clientip_policy ON c = client_ip` --> incorporate client_ip as c.
Or if you'd like to specify the qualifier, first add it to the policy since otherwise it is considered there isn't any (OR we could default to the policy name).
`ENRICH clientip_policy cp ON c = client_ip | KEEP cp.client_ip`

Reply:
Ah, I can see how this can be confusing; `ON client_ip AS c` looks like we're renaming `client_ip`, which wasn't the intention.

I'll go back to `ENRICH clientip_policy c ON client_ip`.